packages/client/src/components/Question/PreQuestionMultipleCheckers.tsx

```diff
@@ -66,15 +67,16 @@ const PreQuestionMultipleCheckers: FunctionComponent<
   return (
     <StepByStepItem
       active={isCurrentQuestion}
+      activeStyle={isCurrentQuestion}
```

Comment:
The prop-name activeStyle seems like it's a style-object but here it's value is a bool. 🤔

Reply:
Nice catch, I will update